docs/sources/api-response-examples/microsoft-365/directory/sanitized-no-app-ids/user.json

```diff
@@ -6,12 +6,12 @@
   "officeLocation":"12/1110",
   "preferredLanguage":"en-US",
   "userPrincipalName":"{\"scope\":\"email\",\"domain\":\"M365x214355.onmicrosoft.com\",\"hash\":\".tbhph3DloGSd5l9e9TeFqmEboAqU8o6BdigmHZOmxI\"}",
-  "id":"p~SIoJOpeSgYF7YUPQ28IWZexVuHyN9A80SJXrbfawKpDRcddGnKI4QDKyjQI9KtjJZDb8FZ27UE_toS68FyWz7Y22fnQYLP91SHJGVwQiN3E",
+  "id":"{\"scope\":\"azure-ad\",\"hash\":\"J0_C7jfy9KhwcfmhAjVNQOnzUnYaE3Kovzt2FAkc2W0\",\"reversible\":\"p~SIoJOpeSgYF7YUPQ28IWZexVuHyN9A80SJXrbfawKpDRcddGnKI4QDKyjQI9KtjJZDb8FZ27UE_toS68FyWz7Y22fnQYLP91SHJGVwQiN3E\"}",
   "otherMails":[
     "{\"scope\":\"email\",\"domain\":\"gmail.com\",\"hash\":\"XRcSIwOXlgrxLZOIHxdtE3GCke5TsftXzOm5zmnm36g\"}"
   ],
   "proxyAddresses":[
-    "{\"scope\":\"email\",\"domain\":\"M365x214355.onmicrosoft.com\",\"hash\":\".tbhph3DloGSd5l9e9TeFqmEboAqU8o6BdigmHZOmxI\"}",
-    "{\"scope\":\"email\",\"domain\":\"M365x214355.onmicrosoft.com\",\"hash\":\"L282PmjOP7xZpTtW7M1sKbWHzVvzLV4Yfp.7x1r3Yk0\"}"
+    "SMTP:t~.tbhph3DloGSd5l9e9TeFqmEboAqU8o6BdigmHZOmxI@M365x214355.onmicrosoft.com",
+    "smtp:t~L282PmjOP7xZpTtW7M1sKbWHzVvzLV4Yfp.7x1r3Yk0@M365x214355.onmicrosoft.com"
```

Comment:
this just looks better; it's correct, bc URL_SAFE_TOKEN encoding of pseudonym without `reversible`; this is compatible with `LEGACY` implementation of pseudonyms